### PR TITLE
feat: update dropdown styling

### PR DESCRIPTION
**Pull request by jake-bassett**

### CODE REVIEW

File: projects/components/src/filtering/filters/numeric/numeric-filter-content-renderer.component.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { ChangeDetectionStrategy, Component, Inject } from '@angular/core';
 import { FilterOperatorType } from '../../model/filter-operator-type';
-import { FILTER_RENDERER_API, FilterRendererApi } from '../../rendering/filter-renderer.component';
+import { FilterRendererApi, FILTER_RENDERER_API } from '../../rendering/filter-renderer.component';
```
Comment (aaron-steinfeld) [inline projects/components/src/filtering/filters/numeric/numeric-filter-content-renderer.component.ts:3]: Hmm do we not lint test files? How'd this get by CI

Comment (aaron-steinfeld) [reply projects/components/src/filtering/filters/numeric/numeric-filter-content-renderer.component.ts:3]: this whole directory is defunct, it was replaced by filter bar. Will remove it in a follow up.

File: projects/components/src/time-range/time-range.component.scss [hunk 2]
```diff
@@ -43,11 +44,10 @@
 }
 
 .popover-content {
-  background: white;
-  border: $gray-1 solid 1px;
+  @include dropdown();
 
   .predefined {
-    width: 197px;
+    width: 198px;
```
Comment (aaron-steinfeld) [inline projects/components/src/time-range/time-range.component.scss:50]: 🎉

File: projects/assets-library/assets/styles/_font.scss [hunk 3]
```diff
@@ -101,36 +101,16 @@ $font-family: 'Work Sans', sans-serif;
...
-  &:hover {
-    background: $gray-1;
-    cursor: pointer;
+    @if ($isSvg) {
+      fill: currentColor;
+    }
   }
 }
 
-@mixin card-hover {
```
Comment (anandtiwary) [inline projects/assets-library/assets/styles/_font.scss:124]: Why are we removing card hover?

Comment (anandtiwary) [reply projects/assets-library/assets/styles/_font.scss:124]: Ignore

File: projects/assets-library/assets/styles/_layout.scss [hunk 4]
```diff
@@ -36,10 +36,39 @@
...
+}
+
+@mixin list-hover {
+  &:hover {
+    background: $gray-1;
+    cursor: pointer;
+  }
+}
+
+@mixin card-hover {
```
Comment (anandtiwary) [inline projects/assets-library/assets/styles/_layout.scss:50]: Do they really belong to layout? May be `_interaction.scss`?

File: projects/components/src/time-range/time-range.component.scss (hunk 2 above)
Comment (anandtiwary) [reply projects/components/src/time-range/time-range.component.scss:50]: hah.

File: projects/assets-library/assets/styles/_layout.scss (hunk 4 above)
Comment (jake-bassett) [reply projects/assets-library/assets/styles/_layout.scss:50]: Yeah, I almost through about creating another file. These certainly didn't fit well in 'fonts', but only fit slightly better in 'layouts'. We can continue to divide these up into additional files as they grow.

Comment (anandtiwary) [reply projects/assets-library/assets/styles/_layout.scss:50]: Can we do it now?  these mixins in layout are confusing to me.

Comment (jake-bassett) [reply projects/assets-library/assets/styles/_layout.scss:50]: I will in a follow up. I have a few branding/styling tickets to follow this one. I'll change it in one of them.

**Pull request merged**